helm/chart/router/README.md

```diff
@@ -81,7 +81,7 @@ helm show values oci://ghcr.io/apollographql/helm-charts/router
 | resources | object | `{}` |  |
 | restartPolicy | string | `"Always"` | Sets the restart policy of pods |
 | rollingUpdate | object | `{}` | Sets the [rolling update strategy parameters](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#rolling-update-deployment).  Can take absolute values or % values. |
-| router | object | `{"args":["--hot-reload"],"configuration":{"health_check":{"listen":"0.0.0.0:8088"},"supergraph":{"listen":"0.0.0.0:4000"}}}` | See https://www.apollographql.com/docs/router/configuration/overview/#yaml-config-file for yaml structure |
+| router | object | `{"args":["--hot-reload"],"configuration":{"health_check":{"listen":"0.0.0.0:8088"},"supergraph":{"listen":"0.0.0.0:4000"}}}` | See https://www.apollographql.com/docs/graphos/reference/router/configuration#yaml-config-file for yaml structure |
```

Comment:
There's an auto-redirect for the old URL, but do we know if there are any other links that need to be updated for this doc change?

(Non-blocking for this PR, just mostly a note for future us)

Reply:
Not sure how this was updated, it must be something automatic.